app/models/user.rb

```diff
@@ -77,4 +77,8 @@ def request_admin_status(organisation_id)
   def self.purge_deleted_users_where(query)
     User.deleted.delete_all(query)
   end
+
+  def superadmin_emails
```

Comment:
based on the purpose of refactoring this should be a class method, and I guess the `user_klass` variable should be removed.